src/lib/viewers/controls/settings/SettingsDropdown.tsx

```diff
@@ -74,52 +71,69 @@ export default function SettingsDropdown<V extends Value = string>({
 
     useClickOutside(dropdownElRef, () => setIsOpen(false));
 
+    // Customize the forwarded ref to combine usage with the ref internal to this component
+    React.useImperativeHandle(
+        ref,
+        () => ({
+            focus: (): void => {
+                if (buttonElRef.current) {
+                    buttonElRef.current.focus();
+                }
+            },
+        }),
+        [],
+    );
+
     return (
-        <div ref={dropdownElRef} className={classNames('bp-SettingsDropdown', className)}>
+        <div className={classNames('bp-SettingsDropdown', className)}>
             <div className="bp-SettingsDropdown-label" id={`${id}-label`}>
                 {label}
             </div>
-            <button
-                ref={buttonElRef}
-                aria-expanded={isOpen}
-                aria-haspopup="listbox"
-                aria-labelledby={`${id}-label ${id}-button`}
-                className={classNames('bp-SettingsDropdown-button', { 'bp-is-open': isOpen })}
-                id={`${id}-button`}
-                onClick={(): void => setIsOpen(!isOpen)}
-                type="button"
-            >
-                {value}
-            </button>
-            <SettingsFlyout className="bp-SettingsDropdown-flyout" isOpen={isOpen}>
-                <SettingsList
-                    ref={listElRef}
-                    aria-labelledby={`${id}-label`}
-                    className="bp-SettingsDropdown-list"
-                    isActive
-                    onKeyDown={handleKeyDown}
-                    role="listbox"
-                    tabIndex={-1}
+            <div ref={dropdownElRef} className="bp-SettingsDropdown-container">
```

Comment:
We should try to avoid the use of `container`, since it can be confusing in SUIT naming. How about `bp-SettingsDropdown-toggle`?